src/components/AddItem.js

```diff
@@ -1,24 +1,83 @@
 import { collection, addDoc } from 'firebase/firestore';
 import { db } from '../lib/firebase';
+import { useState } from 'react';
+// When merging to main with Issue #3, we can remove setUser below
+import { setUser, getUser } from '../storage-utils/storage-utils';
+// When merging to main with Issue #3, we can remove the import below
+import { getToken } from '@the-collab-lab/shopping-list-utils';
 
 export default function AddItem() {
-  const submitHandler = async () => {
+  const [itemName, setItemName] = useState('');
+  const [frequency, setFrequency] = useState(7);
+
+  const submitHandler = async (e) => {
+    e.preventDefault();
+
+    // Lines 17 - 18 are for testing purposes. Once we merge into main with Issue #3, we can remove these two lines
+    const newUserToken = getToken();
+    setUser(newUserToken);
+    // This will remain:
+    const userToken = await getUser();
+    console.log('GET USER TOKEN FROM LS: ', userToken);
```

Comment:
Nitpick: We should avoid deploying console.log statements to production as they can proliferate easily. Let's be sure to clean them up.